results/sentence-transformers__all-MiniLM-L6-v2/OnlineStoreReviewSentimentClassification.json

```diff
@@ -0,0 +1,53 @@
+{
```

Comment:
I think you ran the wrong model here, it should be: [sentence-transformers/paraphrase-multilingual-MiniLM-L12-v2 ](https://huggingface.co/sentence-transformers/paraphrase-multilingual-MiniLM-L12-v2)